chain/network/src/routing.rs

```diff
@@ -367,6 +387,79 @@ impl RoutingTable {
         })
     }
 
+    /// If peer_id is not on memory check if it is on disk in bring it back on memory.
+    fn touch(&mut self, peer_id: &PeerId) {
+        if peer_id == self.peer_id() || self.peer_last_time_reachable.contains_key(peer_id) {
+            return;
+        }
+
+        let me = self.peer_id().clone();
+
+        if let Ok(Some(nonce)) =
+            self.store.get_ser::<u64>(COL_PEER_COMPONENT, Vec::from(peer_id.clone()).as_ref())
+        {
+            let mut update = self.store.store_update();
+            let enc_nonce = u64_as_bytes(nonce);
```

Comment:
We have a utility function that does this conversion in `primitives::utils`. No need to reinvent the wheel.

Reply:
thanks, solved